### PR TITLE
removed badger implementation of db #1394

### PR DESCRIPTION
**Pull request by koseoyoung**
removed badger implementation of db 
works on #1394

### CODE REVIEW

File: db/db.go [hunk 1]
```diff
@@ -126,8 +126,5 @@ func (m *memKVStore) Commit(b KVStoreBatch) (e error) {
 
 // NewOnDiskDB instantiates an on-disk KV store
 func NewOnDiskDB(cfg config.DB) KVStore {
```
Comment (zjshen14) [inline db/db.go]: There's no need to have `NewOnDiskDB`. People should instantiate BoltDB explicitly.

File: db/db_test.go [hunk 2]
```diff
@@ -104,11 +97,7 @@ func TestBatchRollback(t *testing.T) {
 		if kvboltDB, ok := kvStore.(*boltDB); ok {
 			err = kvboltDB.batchPutForceFail(bucket1, testK1[:], testV[:])
 			assert.NotNil(err)
-		} else if kvbadgerDB, ok := kvStore.(*boltDB); ok {
```
Comment (zjshen14) [inline db/db_test.go]: No need to have this if/else logic, as we only test against boltdb impl

Comment (zjshen14) [reply db/db_test.go]: Also you could give up the internal func `NewOnDiskDB`

File: db/db.go (hunk 1 above)
Comment (koseoyoung) [reply db/db.go]: NewOnDiskDB function is now used in other files, so I think it is little bit tricky to change it whole stuff

File: db/db_test.go (hunk 2 above)
Comment (koseoyoung) [reply db/db_test.go]: I'll get rid of if logic

File: consensus/scheme/rolldpos/rolldposctx.go [hunk 3]
```diff
@@ -38,6 +38,14 @@ var (
 		[]string{},
 	)
 
+	consensusHeightMtc = prometheus.NewGaugeVec(
```
Comment (zjshen14) [inline consensus/scheme/rolldpos/rolldposctx.go]: Irrelevant?

Comment (koseoyoung) [reply consensus/scheme/rolldpos/rolldposctx.go]: sorry I fixed it!

**Pull request merged**